api/sheets/places_in.md

```diff
@@ -6,53 +6,33 @@ parent: Google Sheets
 grand_parent: API
 ---
 
-# Retrieve places contained within another place
+# Retrieving Child Nodes
 
-Given a list of parent [`Place`](https://datacommons.org/browser/Place) [DCIDs](/glossary.html),
-(e.g. any [`State`](https://datacommons.org/browser/State), [`Country`](https://datacommons.org/browser/Country), etc.), return a list of child places
-contained within the specified DCIDs. Only returns children whose place type matches
-the request's `placeType` parameter.
+The `=DCPLACESIN(*dcids*, *placeType*) formula returns lists of child places from a list of parent [Place](https://datacommons.org/browser/Place) [DCIDs](https://docs.datacommons.org/glossary.html), such as [State](https://datacommons.org/browser/State), [Country](https://datacommons.org/browser/Country), and so on. It only returns children with a place type that matches the <code>placeType</code> parameter.
 
-## General information about this endpoint
+> **Note**:
+> Be sure to follow the instructions for Installing and Enabling the Sheets Add-On before using this formula.
 
-**Formula**: `=DCPLACESIN(dcids, placeType)`
+## Formula {#formula}
```

Comment:
please check the formatting of these shortcuts (or what's in the brackets). here and below.

Reply:
Verified